testing/python/collect.py

```diff
@@ -325,7 +325,7 @@ def test_gen():
         assert len(colitems) == 1
         gencol = colitems[0]
         assert isinstance(gencol, pytest.Generator)
-        pytest.raises(ValueError, "gencol.collect()")
+        pytest.raises(ValueError, gencol.collect)
```

Comment:
to highlight the importance of the call visually i'd propose a with statement in that case

Reply:
if it's ok, I'm going to leave these as is so it's a less noisy diff -- we can clean these up in a followup